stateproof/db.go

```diff
@@ -53,7 +53,7 @@ const (
     	builder BLOB NOT NULL
     )`
 
-	insertBuilderForRound = `INSERT INTO builders (round,builder) VALUES (?,?)`
+	insertOrReplaceBuilderForRound = `INSERT OR REPLACE INTO builders (round,builder) VALUES (?,?)`
```

Comment:
After some thinking, I think it is better to just re-write the builder and not check if there is a diff.
Builder's data is constructed only from the ledger, which is consistent. So I think it is not worth having a non-optimized query.